src/js/letters/reducers/index.js

```diff
@@ -40,34 +39,45 @@ function letters(state = initialState, action) {
         destination: action.data.meta.address,
         lettersAvailable: true
       };
+    }
     case 'GET_LETTERS_FAILURE':
       // We are currently ignoring this; consider removing once we're sure we've handled
       // the various error scenarios
-      return set('lettersAvailable', false, state);
-    case 'GET_BENEFIT_SUMMARY_OPTIONS_SUCCESS':
-      // Create object for which options to include in post for summary letter
-      // Default all options to true so they appear checked in the UI
-      options = mapValues(action.data.data.attributes.benefitInformation, (value) => {
-        // If the value of the benefit option is not false, it means the user
-        // is eligible for that value. Default benefit options they are eligible for
-        // to true. Keep benefit options that have a value of false as false, so
-        // they are sent as false in the request for the pdf.
-        return value !== false;
+      return _.set('lettersAvailable', false, state);
+    case 'GET_BENEFIT_SUMMARY_OPTIONS_SUCCESS': {
+      // Gather all possible displayed options that the user may have toggled on/off.
```

Comment:
At this point they haven't toggled anything though, correct? This happens before the user toggles anything, and is used as a default list of the options that apply to this user that can then be rendered in the UI.